store/hbase/kv.go

```diff
@@ -136,20 +137,32 @@ func (s *hbaseStore) CurrentVersion() (kv.Version, error) {
 type Driver struct {
 }
 
-// Open opens or creates an HBase storage with given dsn, format should be 'zk1,zk2,zk3|tsoaddr:port/tblName'.
-// If tsoAddr is not provided, it will use a local oracle instead.
-func (d Driver) Open(dsn string) (kv.Storage, error) {
+// Open opens or creates an HBase storage with given dataSource.
+//
+// The format of dsn should be 'hbase://zk1,zk2,zk3/dbname?param1=value1&...'
+// which is described in tidb package. Valid params:
+//     tso:   TSO server addr, should be formatted as 'host:port'.
+//     table: hbase table name. Default 'tidb'.
+// If tso is not provided, it will use a local oracle instead. (for test only)
+func (d Driver) Open(dataSource string) (kv.Storage, error) {
 	mc.mu.Lock()
 	defer mc.mu.Unlock()
 
-	if store, ok := mc.cache[dsn]; ok {
+	if store, ok := mc.cache[dataSource]; ok {
```

Comment:
For hbase, if user opens two stores using different hbase table, we should create a new storage rather than using the old one.
But if user opens two stores with same table and different tso, the best result is get an error.
I will reconsider the cache logic.